benchexec/tools/aprove.py

```diff
@@ -22,14 +22,14 @@ class Tool(benchexec.tools.template.BaseTool2):
     Only the binary (jar) distribution of AProVE is supported.
     """
 
-    REQUIRED_PATHS = ["aprove.jar", "AProVE.sh", "bin", "newstrategy.strategy"]
+    REQUIRED_PATHS = ["aprove.jar", "AProVE.sh", "bin", "newstrategy.strategy", "lib", "fake_include"]
```

Comment:
This line would need to be reformatted (you can run `black` on the file or have a look at CI).

Reply:
Thanks! This should be fixed now. Do I have to squash the commits?

Reply:
Thanks, it is fine as it is.